### DISCUSSION TITLE
Discussion div disappearing, but exsisting

### DISCUSSION THREAD

**Discussion by COSHW** [requarks/wiki#4052] [Help / Questions] [answered]
Hello and thank you for your work on this project.

Suddenly, our wiki server started to send unwanted css style class '.comments-container' having display:none inside of html of all existng and new pages, making the discussion div disappear, but leaving 'View Discussion' button, witch moves the scroll to the top of the page.
![image](https://user-images.githubusercontent.com/45237915/120856549-b6524080-c588-11eb-9fc7-10aba57b870d.png)
Page HTML source of a GET request with status 304 Not Modified:
![image](https://user-images.githubusercontent.com/45237915/120856690-e26dc180-c588-11eb-9296-4afe6a746039.png)
Comments are enabled, permissions are granted. If I uncheck the display css inside dev tool, I even can send/view comments.

When testing on a local machine, there was no such issue, having no style tag in html, that modified '.comments-container' class.
![image](https://user-images.githubusercontent.com/45237915/120857629-34fbad80-c58a-11eb-896a-7c2fb6d352d6.png)

Will patiently wait for your help.
Best regards,
Roman

**Comment by NGPixel** [comment, marked as answer]
Did you add custom CSS under Admin > Theme?

**Reply by COSHW** [reply]
> Did you add custom CSS under Admin > Theme?

Yes, I see now. Turns out it wasn't a bug, but just someone messing around. Thanks a lot.

**Comment by COSHW** [comment]
Fixed by deleting custom CSS in Admin > Theme
